keyboards/adalyn/keymaps/vial/rules.mk

```diff
@@ -0,0 +1,4 @@
+VIA_ENABLE = yes
```

Comment:
VIAL is not supported in the main repo, and the vial config and keymap should be removed.